apps/myaccount/src/components/profile/profile.tsx

```diff
@@ -882,8 +933,12 @@ export const Profile: FunctionComponent<ProfileProps> = (props: ProfileProps): R
                         < Grid.Column mobile={ 6 } tablet={ 6 } computer={ 4 } className="first-column">
                             <List.Content>
                                 {
-                                    !commonConfig.userProfilePage.showEmail &&  fieldName.toLowerCase() === "username"
-                                        ? fieldName + "(Email)"
+                                    (
+                                        !commonConfig.userProfilePage.showEmail 
+                                        ||  usernameConfig?.enableValidator==="false"
+                                    )
+                                    &&  fieldName.toLowerCase() === "username"
+                                        ? fieldName + " (Email)"
```

Comment:
Let's improve the readability here. May be like follows,
```suggestion
                                    ((!commonConfig.userProfilePage.showEmail 
                                        || usernameConfig?.enableValidator === "false")
                                    && fieldName.toLowerCase() === "username")
                                        ? fieldName + " (Email)"
```